doc/ref_model/chapters/chapter07.rst

```diff
@@ -1104,10 +1103,9 @@ System hardening
 +-----------------+--------------------------------------------------+-------------------------------------------------+
 | req.sec.gen.002 | The systems part of the cloud infrastructure     | Hardening: CIS Password Policy Guide            |
```

Comment:
This still need some fixes. I've added them as a pr to your pr: https://github.com/Aryan-Singla/RM/pull/1